idea/testData/quickfix/changeSignature/addEnumConstructorParameter3.kt

```diff
@@ -0,0 +1,8 @@
+// "Add parameter to constructor 'Foo'" "true"
+// DISABLE-ERRORS
+enum class Foo(n: Int) {
+    A(1, 2<caret>),
+    B(3),
+    C(),
```

Comment:
What if we write `C(3, 4)` here? Suppose this `C(3, 4)` will be changed to `C(3, 2)`, which is not correct.